### ISSUE TITLE
Problem with seccomp running containerd test suite on ARM

### ISSUE THREAD

**Issue by gianarb** [moby/moby#35782]
```
root@test-runner-arm64:~# docker version
Client:
 Version:      17.11.0-ce-rc3
 API version:  1.34
 Go version:   go1.8.3
 Git commit:   5b4af4f
 Built:        Wed Nov  8 03:04:48 2017
 OS/Arch:      linux/arm64

Server:
 Version:      17.11.0-ce-rc3
 API version:  1.34 (minimum version 1.12)
 Go version:   go1.8.3
 Git commit:   5b4af4f
 Built:        Wed Nov  8 03:02:57 2017
 OS/Arch:      linux/arm64
 Experimental: false
```

```
root@test-runner-arm64:~# uname -a
Linux test-runner-arm64 4.10.0-38-generic #42~16.04.1-Ubuntu SMP Tue Oct 10 16:33:57 UTC 2017 aarch64 aarch64 aarch64 GNU/Linux
```
I am running `contained` (master) test suite inside a container but I get this error:

```
    --- FAIL: TestDiffApply/Basic (0.02s)
        testsuite.go:43: Apply failed at 1: operation not permitted
            copy file range failed
            github.com/containerd/containerd/fs.copyFileContent
                /go/src/github.com/containerd/containerd/fs/copy_linux.go:43
            github.com/containerd/containerd/fs.copyFile
                /go/src/github.com/containerd/containerd/fs/copy.go:119
            github.com/containerd/containerd/fs.copyDirectory
                /go/src/github.com/containerd/containerd/fs/copy.go:76
            github.com/containerd/containerd/fs.copyDirectory
                /go/src/github.com/containerd/containerd/fs/copy.go:63
            github.com/containerd/containerd/fs.CopyDir
                /go/src/github.com/containerd/containerd/fs/copy.go:24
            github.com/containerd/containerd/archive.diffApplier.Apply
                /go/src/github.com/containerd/containerd/archive/tar_test.go:689
            github.com/containerd/containerd/archive.(*diffApplier).Apply
                <autogenerated>:1
            github.com/containerd/containerd/fs/fstest.makeTest.func1
                /go/src/github.com/containerd/containerd/fs/fstest/testsuite.go:41
            testing.tRunner
                /usr/local/go/src/testing/testing.go:746
            runtime.goexit
                /usr/local/go/src/runtime/asm_arm64.s:931
            failed to copy files
```
Using the option `--security-opt seccomp=unconfined` the test suite works.
Here we are @justincormack ! Sorry for my delay reporting this

**Comment by justincormack** [comment]
You need at least libseccomp 2.3 to include that syscall i think, and Ubuntu 16.04.1 was released with a 4.4 kernel while that syscall only went into 4.5, and you are using a newer one. You either need to use a more recent Ubuntu release, or get a newer libseccomp from elsewhere. (I generally don't recommend older distros on arm64 either, Debian stretch is good if you want a longer term release than the other Ubuntu ones until 18.04 is out).

**Comment by thaJeztah** [comment]
Looks like we can close this one?

FWIW, I also opened a tracking issue to add a "minimum version" for seccomp in the docker packages

I'll go ahead and close, but feel free to continue the conversation 👍
